completion/available/github-cli.completion.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+cite "about-completion"
+about-completion "GitHub CLI completion"
+
+if command -v which gh >/dev/null 2>&1; then
```

Comment:
Let's use `if _command_exists gh ; then` here.

The `_command_exists` function is part of Bash-it's helper functions - it could also help with the `bash-it doctor` function that is discussed in #1501.